src/js/formapp.js

```diff
@@ -127,6 +128,8 @@ async function renderForm({ definition, isReadOnly, storedSubmission, formData,
     // Always run one last change event on submit
     onChangeDebounce.cancel();
     onChange(form, changeReducers);
+    updateSubmissionDebounce.cancel();
```

Comment:
When would this be needed? Out of curiosity.

When a user submits the form before the `updateSubmissionDebounce()` finishes? To avoid a scenario where `router.request('update:storedSubmission', prevSubmission);` is called after a user hits the button to submit the form?

Reply:
so `onChange` calls `updateSubmissionDebounce` now, but at this point it's never really wanted.  I suppose if you have an error _maybe_ it should run, but realistically we never want it to resolve after the form submission, because the form submission deletes the entry and if there's an error, the user is almost for sure going to make a change...  very low cost for any data loss.

Reply:
Okay gotcha, thanks 👍